File: ios/Kurir/modules/login/LoginView.swift
//
//  LoginView.swift
//  Kurir
//
//  Created by Loren on 9/9/19.
//  Copyright © 2019 Crazybean Studio. All rights reserved.
//

import Foundation
import Mobilex

protocol LoginView {
    func showError(auth: Auth?)
    func showNameError()
    func showPasswordError()
    func showDashboard()
    func showNotFound()
    func dismiss()
    func showSpinner()
    func hideSpinner()
}

File: android/src/main/java/au/com/crazybean/mobilex/kurir/modules/auth/login/LoginView.kt
package au.com.crazybean.mobilex.kurir.modules.auth.login

import au.com.crazybean.mobilex.kurir.modules.base.View

interface LoginView : View {
    fun showRegister()
    fun showNameError()
    fun showPasswordError()
    fun showDashboard()
    fun showNotFound()
    fun showSpinner()
    fun hideSpinner()
}